README.md

```diff
@@ -26,11 +26,13 @@ docker-compose up -d postgresdb redis
 # 4) create base .env file
 cp .env.example .env
 
-# 5) create databases for all apps and apps/.env files
+# 5) prepare local packages
+yarn build
```

Comment:
I don't think this is the right fix. Suggested in issues solution was workaround, not TODO.